faq.rst

```diff
@@ -181,8 +183,8 @@ best to run a ``make distclean`` to ensure that all previously built files
 are removed.
 
 
-I want to keep a separate working copy per development branch, is it possible?
-''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+I want to keep a separate working copy per development branch, is it's possible?
```

Comment:
We might also suggest `git worktree` here, I've used it some and it works pretty well.  Equivalent to `hg share`.